### ISSUE TITLE
Implement autobump and autobuild workflow to automatically compile from latest Gecko

### ISSUE THREAD

**Issue by GunGunGun** [CookieJarApps/SmartCookieWeb-Preview#135]
Thanks for developing a game changing Firefox fork, SmartCookieWeb Preview is by far the most innovative Android fork.

But we're usually late when it comes to version update, I think implementing Github workflow for this is worthwhile.

**Comment by CookieJarApps** [comment]
Yes this should be possible, the biggest issues at the moment are that the Mozilla Android Components API is pretty unstable so manual changes are needed with almost every update (though for updates within a major version or if there are no issues this could work), and also some sort of automated testing would probably need to be added even if there's a manual check before the update goes live in case there's a difficult to notice issue. I'll start looking into doing this though.

**Comment by GunGunGun** [comment]
> Yes this should be possible, the biggest issues at the moment are that the Mozilla Android Components API is pretty unstable so manual changes are needed with almost every update (though for updates within a major version or if there are no issues this could work), and also some sort of automated testing would probably need to be added even if there's a manual check before the update goes live in case there's a difficult to notice issue. I'll start looking into doing this though.

Thanks very much.

Just anonther unrelated question, does SCW have a subreddit dedicated to it, people are showing a lot of interest about it lately, if not I hope you consider making one.

[SmartCookieWeb Preview is superior Firefox Android](https://old.reddit.com/r/browsers/comments/1epea2i/smartcookieweb_preview_is_superior_firefox_android/)

[Smart cookie browser](https://old.reddit.com/r/browsers/comments/1evtkla/smart_cookie_browser/)

[SmartCookieWeb-Preview (Android Firefox Fork That Isn't Terrible and Annoying) Version 26 - Out Now!](https://old.reddit.com/r/browsers/comments/1evwr8f/smartcookiewebpreview_android_firefox_fork_that/)

The moderator of /r/browsers community is open to add SCW subreddit to the sidebar to advertise it.
